Marlin/language_da.h

```diff
@@ -190,56 +190,56 @@
 
 
 #if LCD_WIDTH > 19
-#define MSG_INFO_PRINT_COUNT              "Ant. Prints"
-#define MSG_INFO_COMPLETED_PRINTS         "Færdige"
-#define MSG_INFO_PRINT_TIME               "Total print tid"
-#define MSG_INFO_PRINT_LONGEST            "Længste print"
-#define MSG_INFO_PRINT_FILAMENT           "Total Extruderet"
+ #define MSG_INFO_PRINT_COUNT              "Ant. Prints"
+ #define MSG_INFO_COMPLETED_PRINTS         "Færdige"
+ #define MSG_INFO_PRINT_TIME               "Total print tid"
+ #define MSG_INFO_PRINT_LONGEST            "Længste print"
+ #define MSG_INFO_PRINT_FILAMENT           "Total Extruderet"
```

Comment:
I could not find them in the english file which i have copied off

But will add them if i dont hear anything else